rector.php

```diff
@@ -35,7 +35,7 @@
         __DIR__ . '/build/build-preload.php',
     ])
     ->withRootFiles()
-    ->withImportNames(removeUnusedImports: true)
+    ->withImportNames()
```

Comment:
It seems you were testing speed? I think this can be re-enabled again, it already fast on rector-src, and this also enabled in other rector-* packages rector.php.

Reply:
I thought the `removeUnusedImports: true` is enabled by default.

I think we should enable it, so default behavior is the one with least work. What do you think?

Reply:
imo, current optional implementation already correct, it should be optional, as it "remove things" that can possibly there on purpose.

Reply:
I see, I'll restore it here then.